mopidy_musicbox_webclient/static/js/library.js

```diff
@@ -263,7 +266,8 @@ function showArtist(nwuri) {
 //TODO cache
     $('#h_artistname').html('');
     showLoading(true);
-    mopidy.library.lookup(nwuri).then(function(resultArr) {
+    mopidy.library.lookup({'uris': [nwuri]}).then(function(resultDict) {
+        var resultArr = resultDict[Object.keys(resultDict)[0]];
```

Comment:
I got tripped up on how the callbacks were handled in https://github.com/pimusicbox/mopidy-musicbox-webclient/blob/4992d038a950d69a00077ea7e5e80d7b31b72dfa/mopidy_musicbox_webclient/static/js/process_ws.js#L121-L125 and figured that I would just get the URI from the result dictionary every time to be safe.

You're right that it is not necessary here (and in a few other places) and is easier to read. So I've changed it in 4ec547a66ac24561fb1e58f9f9b4f76a031ccf08.